### DISCUSSION TITLE
Handling SEPA direct debit chargebacks

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#4328] [Q&A] [answered]
Hi there, 

we are using the sepa-directdebit plug-in and just got our first chargeback.

How can we change the order status back to unpaid so the real payment status is reflected in Pretix and the amount can be paid using bank transfer?

Thanks, 
Thilo

**Comment by pc-coholic** [comment, marked as answer]
Hi Thilo,

at this point, the only way to do this is to manually record a refund, reverting the ticket order to pending or cancelled state.

If you were to fiddle around with the API, you could also create the refunds with a source-value of `external` - which would not really change much from a technical point of view, but display what happened more clearly.

**Comment by ginkel** [comment]
Thanks, that worked half way. I refunded the full amount, set the order to unpaid and added an item for the chargeback fee. Then I re-generated the invoice. The updated invoice still carries the text:

-- 8< --
Wir werden den vollen Bestellbetrag am oder kurz nach dem 24.07.2024 von deinem Bankkonto
XXXX **** einziehen. Diese Zahlung wird auf deinem Kontoauszug als ***
mit Mandatsreferenz *** und Gläubiger-ID ***
erscheinen.

Wir bitten um Bezahlung des offenen Betrags bis zum 08.08.2024.
-- 8< --

...which is IMHO somewhat misleading. Is there a way to change the payment method to bank transfer?

**Reply by pc-coholic** [reply]
You would need to manually start a payment from the customer-view using the bank-transfer method before cancelling/regenerating the invoice, sorry.

**Reply by ginkel** [reply]
Works like a charm, thank you very much! :-)
